app/models/event.rb

```diff
@@ -5,6 +5,7 @@ class Event < ActiveRecord::Base
   has_many :ticket_types, dependent: :destroy
   has_many :event_staffs, dependent: :destroy
   has_many :highlights, dependent: :destroy
+  has_one :recurring_event, dependent: :destroy
```

Comment:
for readability purposes... similar associations can be grouped... all `has_many` together .... and etc.